src/advanced/bitcoin-anchoring.md

```diff
@@ -444,10 +440,10 @@ Example of JSON response:
 
 - **payload.blockhash**: the hash of the anchored Exonum block
 - **payload.block_height**: the height of the anchored Exonum block
-- **content.payload.prev_tx_chain**: last tx-id of previous chain of
+- **content.payload.prev_tx_chain**: the last tx-id of the previous chain of
```

Comment:
Please, either put "." in the end of each line in the list or remove them from all the lines